sqld/src/lib.rs

```diff
@@ -149,7 +153,8 @@ pub async fn run_server(config: Config) -> Result<()> {
                 #[cfg(feature = "mwal_backend")]
                 vwal_methods,
             )
-            .await?;
+            .await
+            .map_err(Error::ProxyError)?;
```

Comment:
anyhow is fine here as well: we are initiating the proxy server, and failing here means that the app will exit with an error